src/views/Settings.vue

```diff
@@ -1,175 +1,30 @@
-                    <label class="text-sm block">Select dataset</label>
-                    <select :value="context.datasetId"
-                            class="mb-6"
-                            @change="setDataset(($event.target as HTMLInputElement).value)">
-                        <option v-for="dataset in datasets" :key="dataset.datasetId" :value="dataset.datasetId">
-                            {{ dataset.displayName }} ({{ dataset.datasetId }})
-                        </option>
-                    </select>
-                </div>
-
-                <div>
-                    <button v-if="datasets.length > 1" class="bg-gray-500 text-white" @click="deleteDataset">
-                        Delete selected dataset
-                    </button>
-                </div>
-            </div>
-
-            <hr class="mb-6">
-        </template>
-        <template v-else>
-            <button class="bg-gray-500 text-white" @click="addEmptyDataset">
-                Add new dataset
-            </button>
-        </template>
-
-        <label class="text-sm block">Name</label>
-        <input v-model="context.displayName" type="text" placeholder="Name">
-
-        <label class="text-sm block mt-6">Dataset Id</label>
-        <input v-model="context.datasetId" type="text" placeholder="Dataset id">
-
-        <label class="text-sm  block mt-6">API Key</label>
-        <input v-model="context.apiKey" type="text" placeholder="Api key">
-
-        <label class="text-sm  block mt-6">Language</label>
-        <input v-model="context.language" type="text" placeholder="LanguageCode">
-
-        <label class="text-sm block mt-6">Currency</label>
-        <input v-model="context.currencyCode" type="text" placeholder="CurrencyCode">
-
-        <label class="text-sm block mt-6">Server url</label>
-        <input v-model="context.serverUrl" type="text" placeholder="Server Url">
-
-        <div class="flex mt-4 gap-3">
-            <button class="bg-gray-500 text-white" @click="shareLink">
-                Get share link
-            </button>
-
-            <span v-if="copied" class="ml-4 text-green-600">
-                Copied!
             </span>
         </div>
-
-        <hr class="my-10">
-
-        <div>
-            <button class="" @click="save">
-                Save
-            </button>
-
-            <span v-if="saved" class="ml-4 text-green-600">
-                Settings have been saved.
+        <div :class="(openTab == Tabs.Personalization ? 'bg-white' : 'bg-gray-50') + ' text-black rounded-t cursor-pointer w-36 h-10 flex items-center justify-center text-center'" @click="() => openTab = Tabs.Personalization">
+            <span>
+                Personalization
             </span>
         </div>
     </div>
+    
+    <Settings v-if="openTab === Tabs.Settings" class="bg-white rounded-tr rounded-b"/>
+    <Personalisation v-if="openTab === Tabs.Personalization" class="bg-white rounded-tr rounded-b"/>
```

Comment:
Forslaget her var for at undgå at duplikerer dine classes ved at putte begge components ind i samme div :(